indexer/indexer_tests/indexer_tests.pro

```diff
@@ -4,16 +4,18 @@ CONFIG -= app_bundle
 TEMPLATE = app
 
 ROOT_DIR = ../..
-DEPENDENCIES = indexer platform editor geometry coding base protobuf tomcrypt \
-               succinct opening_hours pugixml stats_client \
+DEPENDENCIES = map search_tests_support generator_tests_support search routing indexer \
+               storage editor generator platform coding geometry base stats_client \
```

Comment:
As far as I know generator depends on indexer, so it must be before indexer in the dependency list.